torch/_dynamo/variables/user_defined.py

```diff
@@ -414,6 +415,18 @@ def call_function(
             )
 
             return tensor_variable
+        elif (
+            inspect.getattr_static(self.value, "__new__", None) in (object.__new__,)
+            and inspect.getattr_static(self.value, "__init__", None)
+            is not torch.nn.Module.__init__
+            and not self.source
+        ):
+            # Instantiate a UserDefinedObject class and call its __init__ method.
+            obj = object_new(self.value)
+            variable = UserDefinedObjectVariable(
+                obj,
+            )
+            return variable.call_method(tx, "__init__", args, kwargs)
```

Comment:
Can we consolidate this into L342 where we handle similar cases?

Reply:
Will do.